server/chat-plugins/poll.ts

```diff
@@ -350,27 +350,33 @@ export const commands: ChatCommands = {
 
 		clearqueue: 'deletequeue',
 		deletequeue(target, room, user, connection, cmd) {
+			const [slotString, roomid, update] = target.split(',').map(i => i.trim());
+			const targetRoom = roomid ? Rooms.search(roomid) : room;
+			if (!targetRoom) return this.errorReply(`Invalid room.`);
+			if (!targetRoom.persist) {
+				return this.errorReply("This command is unavailable in temporary rooms.");
+			}
+			this.room = targetRoom;
 			room = this.requireRoom();
 			this.checkCan('mute', null, room);
-			if (!room.minorActivityQueue) {
+			if (!targetRoom.minorActivityQueue) {
 				return this.errorReply(this.tr`The queue is already empty.`);
 			}
-			if (cmd === 'deletequeue' && room.minorActivityQueue.length !== 1 && !target) {
+			if (cmd === 'deletequeue' && targetRoom.minorActivityQueue.length !== 1 && !target) {
 				return this.parse('/help deletequeue');
 			}
 			if (!target) {
-				room.minorActivityQueue = null;
+				targetRoom.minorActivityQueue = null;
 				this.modlog('CLEARQUEUE');
 				this.sendReply(this.tr`Cleared poll queue.`);
 			} else {
-				const [slotString, roomid, update] = target.split(',');
 				const slot = parseInt(slotString);
 				const curRoom = roomid ? (Rooms.search(roomid) as ChatRoom | GameRoom) : room;
 				if (!curRoom) return this.errorReply(this.tr`Room "${roomid}" not found.`);
 				if (isNaN(slot)) {
 					return this.errorReply(this.tr`Can't delete poll at slot ${slotString} - "${slotString}" is not a number.`);
 				}
-				if (!room.minorActivityQueue[slot - 1]) return this.errorReply(this.tr`There is no poll in queue at slot ${slot}.`);
+				if (!targetRoom.minorActivityQueue[slot - 1]) return this.errorReply(this.tr`There is no poll in queue at slot ${slot}.`);
```

Comment:
```suggestion
				if (!room.minorActivityQueue[slot - 1]) return this.errorReply(this.tr`There is no poll in queue at slot ${slot}.`);
```

Changing this back to `room` like I said above will fix the max-len error,